pulsar-broker/src/main/java/org/apache/pulsar/broker/service/schema/SchemaRegistryServiceImpl.java

```diff
@@ -221,6 +210,35 @@ public SchemaVersion versionFromBytes(byte[] version) {
         return schemaStorage.versionFromBytes(version);
     }
 
+    @Override
+    public void initialize(ServiceConfiguration configuration, SchemaStorage schemaStorage)
+          throws PulsarServerException {
+        try {
+            Map<SchemaType, SchemaCompatibilityCheck> checkers =
+                  getCheckers(configuration.getSchemaRegistryCompatibilityCheckers());
+            this.schemaStorage = schemaStorage;
+            this.compatibilityChecks = checkers;
+            this.clock = Clock.systemUTC();
```

Comment:
The point of taking a `clock` parameter was to facilitate precise tests. We shouldn't be using `Clock.systemUTC()` or it just defeats the testing purpose.

Reply:
updated this